### ISSUE TITLE
/usr/bin/mono could not be spawned

### ISSUE THREAD

**Issue by mutsys** [OmniSharp/omnisharp-atom#707]
Started getting this message very recently along with the following in the atom logs:

```
Uncaught Error: spawn /usr/bin/mono ENOENT
```

I only just started getting this very recently, hadn't gotten it in the past. Seems to be ignoring my `$PATH`.

```
mikerosemacbookpro:~ mrose$ which mono
/usr/local/bin/mono
mikerosemacbookpro:~ mrose$ mono -V
Mono JIT compiler version 4.2.1 (explicit/6dd2d0d Tue Nov 10 18:01:18 EST 2015)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  x86
	Disabled:      none
	Misc:          softdebug
	LLVM:          yes(3.6.0svn-mono-(detached/a173357)
	GC:            sgen
```

I assure you that `/usr/local/bin` is in my `$PATH`. Can't for the life of me figure out just where `/usr/bin` is getting hardcoded into this attempt launch it. Or for that matter, why it doesn't just want to use CoreClr instead of Mono...

**Comment by david-driscoll** [comment]
Are you sure it's omnisharp atom, I know I've seen similar issues reported with ionide.

I'll still check it out.

**Comment by willl** [comment]
Hi @mutsys, if you do a search it seems others have reported [similar issues](https://github.com/OmniSharp/omnisharp-atom/issues?utf8=%E2%9C%93&q=%2Fusr%2Fbin%2Fmono) previously. It is highly probable that another plugin that is causing the issue. 

Eg: #692 was reported but ended up being an issue with the F# linter. 

Perhaps try disabling plugins one by one can help you to isolate which plugin the issue is coming from. It is highly unlikely to be OmniSharp-Atom. 

As for CoreCLR instead of mono? That's already possible if you use one of the [v1.9 alpha releases](https://github.com/OmniSharp/omnisharp-roslyn/releases) of [omnisharp-roslyn](https://github.com/OmniSharp/omnisharp-roslyn).

**Comment by mutsys** [comment]
I assumed it was OmniSharp since the error message in the logs was preceded by a line from OmniSharp. However, I did have ionide installed as well. Removed it and the error message went away.  But now I get a new error message when OmniSharp asks me to choose a solution file for the c# source. I'll review other issues to see if others have reported it before posting a new issue.  Thank you for the speedy response.
